### PR TITLE
fix: don't write from the read function

### PR DESCRIPTION
**Pull request by Stebalien**
This was causing a double-write.

fixes https://github.com/libp2p/go-libp2p-pnet/issues/31
probably fixes https://github.com/ipfs/go-ipfs/issues/6802
fixes https://github.com/ipfs/go-ipfs/issues/6197

### CODE REVIEW

**Comment by Stebalien** [comment]
cc @bonedaddy

**Comment by Stebalien** [comment]
cc @bonedaddy

**Comment by Stebalien** [comment]
After we merge this, we can revert https://github.com/libp2p/go-ws-transport/pull/65.

File: multistream.go [hunk 1]
```diff
@@ -381,29 +381,25 @@ func (msm *MultistreamMuxer) Handle(rwc io.ReadWriteCloser) error {
...
+func ReadNextTokenBytes(r io.Reader) ([]byte, error) {
+	data, err := lpReadBuf(r)
 	switch err {
 	case nil:
 		return data, nil
 	case ErrTooLarge:
-		err := delimWriteBuffered(rw, []byte("messages over 64k are not allowed"))
-		if err != nil {
-			return nil, err
-		}
```
Comment (vyzo) [inline multistream.go]: are we reporting this error anywhere now?

Comment (Stebalien) [reply multistream.go]: Back to the user. We don't bother sending it to the other side. If someone establishes a connection and sends us garbage, we'll reset the connection and walk away.

**Pull request merged**